examples/ex23p.cpp

```diff
@@ -0,0 +1,1047 @@
+//                       MFEM Example 23 - Parallel Version
+//
+// Compile with: make ex23p
+//
+// Sample runs:  mpirun -np 4 ex23p -o 2 -f 1.0 -rs 1 -rp 1 -prob 0
+//               mpirun -np 4 ex23p -o 3 -f 1.0 -rs 1 -rp 1 -prob 1
+//               mpirun -np 4 ex23p -o 2 -f 3.0 -rs 3 -rp 1 -prob 2
+//               mpirun -np 4 ex23p -o 2 -f 1.0 -rs 2 -rp 1 -prob 3
```

Comment:
This is slow. Reducing to `-rs 1` is faster and still accurate enough.

Reply:
fixed